frontend/src/components/AssistedReporting/MedicalBlock.vue.js

```
@@ -0,0 +1,185 @@
+    isActive: Boolean,
+    extraParams: Object,
+    actionLabel: { type: String, default: 'Weiter' },
+    showAction: { type: Boolean, default: true },
+    loading: Boolean
+});
+const isExpanded = ref(props.isActive);
+const expandStyle = computed(() => ({
+    transform: isExpanded.value ? 'rotate(180deg)' : 'rotate(0deg)',
+    transition: 'transform 0.3s ease'
+}));
+debugger; /* PartiallyEnd: #3632/scriptSetup.vue */
```

Biome (2.3.13)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)